pkgs/development/tools/icestorm/default.nix

```diff
@@ -1,19 +1,19 @@
-{ stdenv, fetchFromGitHub, python3, libftdi }:
+{ libcxxStdenv, fetchFromGitHub, python3, libftdi }:
 
-stdenv.mkDerivation rec {
+libcxxStdenv.mkDerivation rec {
```

Comment:
Depends on how involved you are with upstream... For instance, could it be just a little slip up that upstream use "CC = ..." instead of "CC ?= ..."? One might also simply try to patch in "CC = gcc" in the Nix file. Maybe it works?

I think if you use libcxxStdenv, you should make a copy of your above comment, to justify the change.